microsoft/botframework-solutions
# [Botskills] Implement tests and validations for list functionality

**DEPENDS ON** #1369

## Description
- Implement tests for list functionality.
- Implement validations for list functionality.
- Remove unnecessary validations from list functionality.

## Testing Steps
1. Go to `AI\lib\typescript\botskills\`.
1. Open a terminal in that location.
1. Run the command `npm install` to install dependencies.
1. Run the command `npm run build` to build the project.
1. Run the command `npm run test` to execute the tests for the `list` functionality.
1. Check the 5 tests are passing.

## Checklist
